LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017-2020, SBCB Oxford
```

Comment:
I attributed the work to "SBCB Oxford". Any other suggestion is welcome.

Reply:
Not sure, thoughts here @philbiggin ?